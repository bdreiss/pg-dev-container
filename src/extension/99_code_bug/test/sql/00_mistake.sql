-- KO - Null parameter is not supported
SELECT code_mistake(NULL);

-- KO - Value out of INT range
SELECT code_mistake(99999999999999999);

-- KO - Nagative values are not supported
SELECT code_mistake(-100);

SELECT code_mistake(0);

SELECT code_mistake(10);

SELECT code_mistake(100);

