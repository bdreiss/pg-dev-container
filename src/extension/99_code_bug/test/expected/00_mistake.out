-- KO - Null parameter is not supported
SELECT code_mistake(NULL);
 code_mistake 
--------------
 
(1 row)

-- KO - Value out of INT range
SELECT code_mistake(99999999999999999);
ERROR:  function code_mistake(bigint) does not exist
LINE 1: SELECT code_mistake(99999999999999999);
               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
-- KO - Nagative values are not supported
SELECT code_mistake(-100);
ERROR:  max value must be greater than zero
SELECT code_mistake(0);
INFO:  all "1" elements are contaied
 code_mistake 
--------------
 
(1 row)

SELECT code_mistake(10);
INFO:  all "11" elements are contaied
 code_mistake 
--------------
 
(1 row)

SELECT code_mistake(100);
INFO:  all "101" elements are contaied
 code_mistake 
--------------
 
(1 row)

